### PR TITLE
modify CFLAGS_PLATFORM and CPPFLAGS_PLATFORM to support loongarch64

### PR DESCRIPTION
**Pull request by zhangwenlong8911**
Signed-off-by: Zhang Wenlong <zhangwenlong@loongson.cn>

### CODE REVIEW

File: conf/Makefile.common [hunk 1]
```diff
@@ -21,8 +21,8 @@ if COND_powerpc_ieee1275
   CFLAGS_PLATFORM += -mcpu=powerpc
 endif
 if COND_loongarch64
-  CFLAGS_PLATFORM += -fno-strict-aliasing -march=loongarch -mabi=lp64 -fno-plt -Wa,-mla-global-with-pcrel
-  CPPFLAGS_PLATFORM = -fno-strict-aliasing -march=loongarch -mabi=lp64 -fno-plt -Wa,-mla-global-with-pcrel
+  CFLAGS_PLATFORM += -fno-strict-aliasing -march=loongarch64 -mabi=lp64d -fno-plt -Wa,-mla-global-with-pcrel
+  CPPFLAGS_PLATFORM = -fno-strict-aliasing -march=loongarch64 -mabi=lp64d -fno-plt -Wa,-mla-global-with-pcrel
```
Comment (xen0n) [inline conf/Makefile.common]: 确定下用 `lp64d` 而不是 `lp64s`? GRUB 这种软件不应该用到 FPU。

Comment (zhangwenlong8911) [reply conf/Makefile.common]: 我了解了下关于数据模型的知识，应该是lp64s更合理一些，我修改下昨代码

**Comment by yetist** [comment]
最新工具链应该不需要加-march和-mabi参数了，可以直接删掉。

**Comment by xen0n** [comment]
Proper support has been merged (see #5 #6), and 2.12 is upcoming. Closing for now, feel free to re-open if 2.06 support is still wanted here.

**Pull request closed without merging**